services/supervisor/src/proto_structures.rs

```diff
@@ -54,13 +62,41 @@ pub struct StartService {
     pub deadline_height: Height,
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, ProtobufConvert)]
+#[exonum(pb = "proto::ConfigPropose")]
+pub struct ConfigPropose {
+    /// The height until which the update configuration procedure should be completed.
+    pub actual_from: Height,
+    /// New configuration proposition.
+    pub changes: Vec<ConfigChange>,
+}
+
+#[derive(Debug, Clone, PartialEq, ProtobufConvert)]
+#[exonum(pb = "proto::ConfigVote")]
+pub struct ConfigVote {
+    /// Hash of configuration proposition.
+    pub propose_hash: Hash,
+}
+
+/// Pending config change proposal entry
+#[derive(Clone, Debug, Eq, PartialEq, ProtobufConvert, Serialize, Deserialize)]
+#[exonum(pb = "proto::ConfigProposalWithHash")]
+pub struct ConfigProposalWithHash {
```

Comment:
Please add documentation here and `#[warn(missing_docs)]` in the lib.rs to catch issues like this.